docs/guide/migrating-to-version-9.md

```diff
@@ -342,6 +342,7 @@ Kongponents styles are no longer designed to be utilized standalone, separately
 #### Constants, Types & Interfaces
 
 * `TimeRange` - `start` and `end` keys are now defined as `Date | null`
+* `hidePopover` property in `DateTimePickerState` interface has been renamed to `popoverOpen`
```

Comment:
so this will have to be "flipped" in all the host apps right?

Reply:
Yes but I didn't find any references in our repos